docs/flashbots-mev-boost/architecture/MEV-boost-block-proposal.md

```diff
@@ -0,0 +1,10 @@
+## MEV-Boost Block Proposal
+
+![MEV-Boost Block Proposal](https://s3.us-west-2.amazonaws.com/secure.notion-static.com/ba103f23-f255-4424-9efc-7229bc598243/Untitled.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Credential=AKIAT73L2G45EIPT3X45%2F20220830%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20220830T205159Z&X-Amz-Expires=86400&X-Amz-Signature=3ec5ca772e1bdf55a4711056d34f7936fe92ff020aea4128a0e983c7e7a71336&X-Amz-SignedHeaders=host&response-content-disposition=filename%20%3D%22Untitled.png%22&x-id=GetObject)
```

Comment:
image url is not working. maybe use the link  from the image in the github repository readme